codelabs/template.md

```diff
@@ -0,0 +1,56 @@
+author: YOUR NAME
+summary: CODELAB SUMMARY
+id: TEMPLATE-CODELAB-ID
```

Comment:
Is there any kind of format you want for the codelab id? For example do we want Blockly in front of all of them? Or does it matter?

Reply:
I'm not sure yet.  I think it doesn't matter too much since it isn't going to into the more general codelabs list.